docs/en_US/MultiPhase.md

```diff
@@ -48,6 +70,10 @@ trial_end
 ```
 With this information, the tuner could know which trial is requesting a configuration, and which trial is reporting results. This information provides enough flexibility for your tuner to deal with different trials and different phases. For example, you may want to use the trial_job_id parameter of generate_parameters method to generate hyperparameters for a specific trial job.
 
-Of course, to use your multi-phase tuner, __you should add `multiPhase: true` in your experiment YAML configure file__.
 
-[ENAS tuner](https://github.com/countif/enas_nni/blob/master/nni/examples/tuners/enas/nni_controller_ptb.py) is an example of a multi-phase tuner.
+### Tuners support multi-phase experiments:
+
+[TEP](BuiltinTuner.md#TPE), [Random](BuiltinTuner.md#Random), [Anneal](BuiltinTuner.md#Anneal), [Evolution](BuiltinTuner.md#Evolution), [SMAC](BuiltinTuner.md#SMAC), [NetworkMorphism](BuiltinTuner.md#NetworkMorphism), [MetisTuner](BuiltinTuner.md#MetisTuner), [ENAS tuner](https://github.com/countif/enas_nni/blob/master/nni/examples/tuners/enas/nni_controller_ptb.py).
```

Comment:
BOHB and Hyperband?

Reply:
let me work on it today :)

Reply:
the pr that supporting multi-phase on BOHB and Hyperband is merge, please update the doc accordingly.

Reply:
Thanks, updated.